plugins/reference-assets/core/src/assets/choice/transform.ts

```diff
@@ -0,0 +1,33 @@
+import type { TransformFunction } from '@player-ui/player';
+import type { ChoiceAsset, TransformedChoice } from './types';
+import { choicesEntryTransform } from './choices-entry'
+
+/**
+ * Choice asset transform
+ */
+export const choiceTransform: TransformFunction<ChoiceAsset, TransformedChoice> = (
+  asset,
+  options
+) => {
+  return {
```

Comment:
It seems like you're limiting your options by trying to keep this transform to a single `return` statement